recipe/meta.yaml

```diff
@@ -1,54 +1,76 @@
-{% set version = "3.141.0" %}
+{% set version = "4.24.0" %}
 
 package:
   name: selenium
   version: {{ version }}
-
+#  Added 0000-no-rust-setup.patch to remove all references to Rust
+#  extensions from setup.py. This includes removing the import of setuptools_rust and
+#  the rust_extensions configuration, which resolves build failures caused by the unnecessary
```

Comment:
What was the failure message? Why does it fails with setuptools-rust?

Reply:
In tandem with `setuptools-rust`, there is an error in which `wheel` is unable to be built successfully. Conda-forge also experienced this issue and implemented this patch as well. See [here ](https://github.com/AnacondaRecipes/selenium-feedstock/pull/5/commits/847d45111034bb6fbe78852c3722a91b7011a0b3)for logs regarding this issue.

Reply:
The error is:
`error: can't find Rust compiler`

So you should add the Rust compiler to the `build` section: 
  - {{ compiler('rust') }}

And enable `setuptools-rust` again in `host`

Reply:
I fixed it and pushed my changes